source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -73,7 +73,7 @@ An "applicationError" object has the following keys:
   - "timeout": A network timeout error.
 
 - response: (optional) A command error response, for example
-  ``{ok: 0, errmsg: "not master"}``. Present if and only if ``type`` is
+  ``{ok: 0, errmsg: "not writable primary"}``. Present if and only if ``type`` is
```

Comment:
Should this remain as-is? I may be misremembering. Does the server replace this error message with "not writable primary" if the client connects performs the handshake with `hello`?

Reply:
As soon as you run `hello` on a connection, the error message changes from `not master` to `not primary`. The error code remains 10107 and `NotWritablePrimary` regardless. Only `errmsg` changes based on whether `hello` has been called on that connection.

I'll fix the text to make mention of this fact and correct error message text.